openassessment/xblock/grade_mixin.py

```diff
@@ -195,6 +198,27 @@ def _is_incomplete(step):
             {'incomplete_steps': incomplete_steps}
         )
 
+    def render_grade_override(self, override_score):
+        """
+        Render the override points and points possible.
+
+        Args:
+            override_score (dict): Part of the serialized Workflow model.
+
+        Returns:
+            tuple of template path (string), context (dict)
+        """
+        score = {}
+        score['points_possible'] = override_score['points_possible']
+        score['override_score'] = override_score['points_earned']
+        context = {
+            'score': score,
```

Comment:
Since we're creating the entire object here on-the-fly, can we just do so inline? like:

```python
context = {
    'score': {
        'points_possible': override_score['points_possible'],
        'override_score': override_score['override_score'],
    },
}